CONTRIBUTING.md

```diff
@@ -0,0 +1,223 @@
+# Contributing to Atlas CLI PLUGIN KUBERNETES
```

Comment:
```suggestion
# Contributing to Atlas CLI Kubernetes Plugin 
```

Reply:
[q] wonder if we should call it Kubernetes Plugin everywhere despite the repo name

Reply:
I agree it sounds better as 'Kubernetes Plugin'. And its not so big a change to suggest we are referring to some other repo.

Reply:
Done!